apps/students/views.py

```diff
@@ -36,7 +36,11 @@
     login_url=reverse_lazy('accounts:complete_profile')
 )
 def index(request):
-    return HttpResponse('Hello there!')
+    students = Student.objects.order_by('village','school_class','first_name','last_name')
```

Comment:
We don't need to sort by village, rest are fine. We can later add a filter by village.

Reply:
removes sorting by village..